### PR TITLE
Update to v3.3.3

### PR DESCRIPTION
**Pull request by renich**

### CODE REVIEW

File: com.bitwig.BitwigStudio.yaml [hunk 1]
```diff
@@ -29,6 +29,9 @@ finish-args:
   # For desktop notifications
   - --talk-name=org.freedesktop.Notifications
 
+  # For secrets communication
+  - --talk-name=org.freedesktop.secrets
```
Comment (abique) [inline com.bitwig.BitwigStudio.yaml:33]: Why?
Is it useful for accessing network folders which requires authentication?
If you could give an example of why you need it please :)

**Review by abique — COMMENTED** [review]
Please can you explain me why you added --talk-name=org.freedesktop.secrets ?

File: com.bitwig.BitwigStudio.yaml (hunk 1 above)
Comment (abique) [reply com.bitwig.BitwigStudio.yaml:33]: Ah is this for vital?

Comment (renich) [reply com.bitwig.BitwigStudio.yaml:33]: ah, yes. I was working on that. I'll remove it for now.

**Pull request merged**